doc/version_history.rst

```diff
@@ -1,11 +1,27 @@
-.. py:currentmodule:: lsst.ts.MTMount
+.. py:currentmodule:: lsst.ts.mtmount
 
-.. _lsst.ts.MTMount.version_history:
+.. _lsst.ts.mtmount.version_history:
 
 ###############
 Version History
 ###############
 
+v0.21.0
+-------
+
+Changes:
+
+* Renamed the package from ts_MTMount to ts_mtmount,
+  and the Python namespace from lsst.ts.MTMount to lsst.ts.mtmount.
```

Comment:
A bit picky but shouldn't one line be used per sentence instead of splitting it out over two?

Reply:
The line was long enough that I chose to break it at the comma. I often do that. Technically perhaps I shouldn't.